### PR TITLE
TCD-2792

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/api/insights.md [hunk 1]
```diff
@@ -933,3 +977,3102 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 </details>
 
 ---
+
+### Get Activity For Users
```
Comment (LoryApiFortress) [inline docs/dev/api/insights.md:981]:
```suggestion
### Get Activity for Users
```

**Comment by LoryApiFortress** [comment]
@paweldudzinski  can you please review and:
- add description for each endpoint
- add description for the query params 
Thanks

File: docs/dev/api/insights.md [hunk 2]
```diff
@@ -933,3 +997,2908 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+    <tr>
+     <td><code>end</code></td>
+       <td><p><small>| QUERY | OPTIONAL | DATE |</small></p><p>The ending date of the period during which the test runs executed, in <code>YYYY-MM-DDTHH:mm:ssZ</code> (UTC) format.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>limit</code></td>
+     <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Identifies the number of records to return. Default value is <code>50</code>.</p></td>
+    </tr>
```
Comment (LoryApiFortress) [inline docs/dev/api/insights.md:1579]:
```suggestion
    </tr>
```

File: docs/dev/api/insights.md [hunk 3]
```diff
@@ -933,3 +997,2908 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>offset</code></td>
+     <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Specifies the number of items to be skipped from the beginning of the list. Default value is <code>0</code>.</p></td>
```
Comment (LoryApiFortress) [inline docs/dev/api/insights.md:1584]:
```suggestion
     <td><p><small>| QUERY | OPTIONAL | INTEGER |</small></p><p>Specifies the number of items to be skipped from the beginning of the list. Default value is <code>0</code>.</p></td>
```

File: docs/dev/api/insights.md [hunk 4]
```diff
@@ -933,3 +997,2908 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+  <tbody>
+    <tr>
+     <td><code>coverage_field</code></td>
+       <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Available values are:<code>device</code>, <code>browser</code>, <code>os</code>.</p><p>
+     </p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>automation_backend</code></td>
```
Comment (LoryApiFortress) [inline docs/dev/api/insights.md:3082]:
```suggestion
     <td><code>automation_backend</code></td>
```

File: docs/dev/api/insights.md [hunk 5]
```diff
@@ -154,54 +156,51 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
-      "end_time": "2017-03-01T12:16:22Z",
-      "status": "complete",
-      "error": "",
-      "os": "OS X El Capitan (10.11)",
-      "browser": "iPhone 10.0",
-      "details_url": "https://saucelabs.com/rest/v1.1/a-team/jobs/50fdd2369eer3031ad912kib57b12440"
-    },
-    // 9 more tests
+      "ancestor": "<ancestor>",
+      "browser": "<broswer>",
```
Comment (aliciakrebs) [inline docs/dev/api/insights.md:171]:
```suggestion
      "browser": "<browser>",
```

File: docs/dev/api/insights.md [hunk 6]
```diff
@@ -502,84 +519,131 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+            "name": "<browser>",
+            "count": 3
+          }
+        ],
+        "browserError": [
           {
-            "name": "Windows 7",
-            "count": 65
-          },
+            "name": "<broswer>",
```
Comment (aliciakrebs) [inline docs/dev/api/insights.md:547]:
```suggestion
            "name": "<browser>",
```

File: docs/dev/api/insights.md [hunk 7]
```diff
@@ -933,3 +997,2908 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+          "creation_time": "2023-04-18T21:14:29.374Z",
+          "start_time": "2023-04-18T21:14:29.374Z",
+          "end_time": "2023-04-18T21:14:29.374Z",
+          "duration": 4,
+          "status": "complete",
+          "error": "<error>",
+          "os": "<os>",
+          "os_normalized": "<os>",
+          "browser": "<browser>",
+          "browser_normalized": "<broswer>",
```
Comment (aliciakrebs) [inline docs/dev/api/insights.md:3652]:
```suggestion
          "browser_normalized": "<browser>",
```

File: docs/dev/api/insights.md [hunk 8]
```diff
@@ -933,3 +997,2908 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+            "creation_time": "2023-04-18T21:32:45.153Z",
+            "start_time": "2023-04-18T21:32:45.153Z",
+            "end_time": "2023-04-18T21:32:45.153Z",
+            "duration": 9,
+            "status": "passed",
+            "error": "<error>",
+            "os": "<os>",
+            "os_normalized": "<os>",
+            "browser": "<browser>",
+            "browser_normalized": "<broswer>",
```
Comment (aliciakrebs) [inline docs/dev/api/insights.md:3856]:
```suggestion
            "browser_normalized": "<browser>",
```

File: docs/dev/api/insights.md [hunk 9]
```diff
@@ -933,3 +997,2908 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
...
+        "build": "build456",
+        "creation_time": "2023-04-18T21:32:45.153Z",
+        "start_time": "2023-04-18T21:32:45.153Z",
+        "end_time": "2023-04-18T21:32:45.153Z",
+        "duration": 10,
+        "status": "failed",
+        "error": "<error>",
+        "os": "<os>",
+        "os_normalized": "<os>",
+        "browser": "<broswer>",
```
Comment (aliciakrebs) [inline docs/dev/api/insights.md:3890]:
```suggestion
        "browser": "<browser>",
```

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**